### ISSUE TITLE
error resolving plugin - 404

### ISSUE THREAD

**Issue by delta1** [dprint/check#8]
Heya, this action has started failing with the following error 

```
Error resolving plugin https://plugins.dprint.dev/rustfmt-0.6.1.exe-plugin@99b89a0599fd3a63e597e03436862157901f3facae2f0c2fbd0b9f656cdbc2a5: 

Error downloading https://plugins.dprint.dev/rustfmt-0.6.1.exe-plugin@99b89a0599fd3a63e597e03436862157901f3facae2f0c2fbd0b9f656cdbc2a5 - 404 Not Found 
``` 
Trying this link does indicate 404 

https://plugins.dprint.dev/rustfmt-0.6.1.exe-plugin@99b89a0599fd3a63e597e03436862157901f3facae2f0c2fbd0b9f656cdbc2a5

**Comment by delta1** [comment]
Seems to have occurred with the release of 0.31, since 0.30.3 is still working.

**Comment by dsherret** [comment]
My bad, I made a change in 0.31 and forgot about the old .exe-plugin extensions (nowadays, dprint uses .json). I committed a fix and am doing a release now. Should be fixed in about 15-30 mins.

**Comment by dsherret** [comment]
Should be fixed now in 0.31.1. Sorry about that again!

**Comment by delta1** [comment]
Thanks @dsherret ! Testing now

**Comment by delta1** [comment]
Confirmed fixed in 0.31.1
